src/precice/config/ParticipantConfiguration.cpp

```diff
@@ -164,8 +164,8 @@ ParticipantConfiguration::ParticipantConfiguration(
                                "processors. Both result in the same distribution (if the safety factor is sufficiently large). "
                                "\"on-master\" is not supported if you use two-level initialization. "
```

Comment:
These filters should be updated too.

Reply:
This filter is updated in https://github.com/precice/precice/pull/1253